ACMILabs/acmi-api
# #28 Remove images, videos, and thumbnails from the Works API

Resolves #28

Remove images and videos from the Works API while we audit the rights we have to include them.

### Acceptance Criteria
- [x] Remove `images` and `thumbnail` from the Works API
- [x] Remove `videos` from the Works API
- [x] Add `INCLUDE_IMAGES` and `INCLUDE_VIDEOS` to [acmi-devops](https://github.com/ACMILabs/acmi-devops/issues/437)
- [x] Remove public read from S3 bucket: `acmi-public-api` in [acmi-devops](https://github.com/ACMILabs/acmi-devops/issues/437)
- [x] Update the README with instructions for retrieving TMDB & IGDB images
- [x] After merging the acmi-devops PR, then this PR, delete `app/json/works` and run a full update
- [x] After merging the acmi-devops PR, then this PR, delete the search index, and run a full index

### Relevant design files
* None

### Testing instructions
1. Set `DEBUG=true` and `UPDATE_WORKS=true` in your `config.env`
1. Delete your local folder `app/json/works/`
1. Start the dev environment: `cd development` and `docker-compose up`
1. Feel free to quit after a few pages have been updated
1. Set `UPDATE_WORKS=false` in your `config.env`
1. Start your dev environment again: `docker-compose up`
1. Note there aren't any `images`, `videos`, or `thumbnail` fields: https://localhost:8081/works/
1. Set `UPDATE_SEARCH=true` in your `config.env`
1. Start your dev environment again: `docker-compose up`
1. Feel free to quit after a few works have been indexed
1. Check the search to note there aren't any `images`, `videos`, or `thumbnail` fields
1. Set `UPDATE_SEARCH=false` and `UPDATE_WORKS=true` in your `config.env`
1. Set `INCLUDE_IMAGES=true` and `INCLUDE_VIDEOS=true` in your `config.env`
1. Start your dev environment again: `docker-compose up`
1. Set `UPDATE_WORKS=false` in your `config.env`
1. Start your dev environment again: `docker-compose up`
1. Note there are `images`, `videos`, and `thumbnail` fields once again: https://localhost:8081/works/

### DoD
For requester to complete:
